data/yaml/venues/cl4health.yaml

```diff
@@ -0,0 +1,2 @@
+acronym: CL4Health
+name: First Workshop on Patient-Oriented Language Processing @LREC-COLING-2024 (CL4Health)
```

Comment:
The venue title should not include "First" or "@LREC-COLING-2024". Please remove this (and add a check to the ingestion script to check for the year or ordinal numbers, which are always a mistake here).

Reply:
Done